mteb/tasks/Retrieval/eng/BrightRetrieval.py

```diff
@@ -0,0 +1,132 @@
+from collections import defaultdict
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskRetrieval, MultilingualTask
+
+DOMAINS = [
+    "biology",
+    "earth_science",
+    "economics",
+    "psychology",
+    "robotics",
+    "stackoverflow",
+    "sustainable_living",
+    "pony",
+    # "leetcode",
+    # "aops",
+    # "theoremqa_theorems",
+    # "theoremqa_questions"
+]
+
+DOMAINS_langs = {domain: [domain] for domain in DOMAINS}
```

Comment:
```suggestion
DOMAINS_langs = {domain: [LANG_CODES_FOR_THE_SPLIT] for split in DOMAINS}
```
The format should be like this